src/components/inputField/InputField.jsx

```diff
@@ -0,0 +1,12 @@
+const InputField = ({type, placeholder, name}) => {
```

Comment:
Ennnn waar is de `onChange` en `value` prop? Anders kunnen we geen form-validatie toepassen!